reference-implementation/lib/writable-stream.js

```diff
@@ -760,6 +782,7 @@ function WritableStreamDefaultControllerGetChunkSize(controller, chunk) {
   try {
     return strategySize(chunk);
   } catch (chunkSizeE) {
+    // TODO: Should we notify the sink of this error?
```

Comment:
I took this comment out because I thought we had consensus not to do anything about this until we have finally().